### ISSUE TITLE
Wrong Btrfs mountpoints parsing when rootfs is on Btrfs also

### ISSUE THREAD

**Issue by igogold** [openmediavault/openmediavault#1519]
**The bug description**

After recent update (6.3.5-2) of core openmediavault package daily cron task generates an error.

I have rootfs on Btrfs partition (and additional mountpoints on the system disk) so the output of `findmnt --all --canonicalize --notruncate --uniq --noheadings --output SOURCE,TARGET --types btrfs` command looks like
```text
/dev/nvme0n1p4[/@rootfs]     /
/dev/nvme0n1p4[/@NAS]        ├─/NAS
/dev/nvme0n1p4[/@.snapshots] ├─/.snapshots
/dev/sda                     └─/srv/dev-disk-by-uuid-fa58418a-7658-484e-8d13-c7feedef4781
```

The parser split device and mountpoint parts by space and catch pseudo graphics chars together with mountpoint dir. The run of `btrf device stats` command then gives an error:
```text
+ btrfs device stats --check ├─/.snapshots
ERROR: cannot check ├─/.snapshots: No such file or directory
ERROR: '├─/.snapshots' is not a mounted btrfs device
```

**To Reproduce**

1. Use Btrfs for rootfs
2. Run `/etc/cron.daily/openmediavault-check_btrfs_errors`
3. See error

**Daily cron email output**

```text
/etc/cron.daily/openmediavault-check_btrfs_errors:
Performing an error check on Btrfs file systems.
Checking file system /dev/nvme0n1p4[/@.snapshots] for errors ...
ERROR: cannot check ├─/.snapshots: No such file or directory
ERROR: '├─/.snapshots' is not a mounted btrfs device
run-parts: /etc/cron.daily/openmediavault-check_btrfs_errors exited with return code 1
/etc/cron.daily/openmediavault-check_locked_users:
Checking for locked/banned users.
INFO: No locked/banned users found.
/etc/cron.daily/openmediavault-check_ssl_cert_expiry:
Perform a check for expired SSL certificates.
/etc/cron.daily/openmediavault-pending_config_changes:
Checking for pending configuration changes.
```

**openmediavault Server:**

- OS version: `Linux nas 6.1.15-1-pve #1 SMP PREEMPT_DYNAMIC PVE 6.1.15-1 (2023-03-08T08:53Z) x86_64 GNU/Linux`
- openmediavault version: `6.3.5-2`
- util-linux version (`findmnt` from it): `2.36.1-8+deb11u1`

**Comment by votdev** [comment]
Fixed yesterday and will be available with `openmediavault 6.3.6`, see https://github.com/openmediavault/openmediavault/commit/9cbc82d21925303ad54f96bc7f25c208ed522280.

**Comment by igogold** [comment]
As I see the fix applied only to `/etc/cron.weekly/openmediavault-scrub_btrfs` but `/etc/cron.daily/openmediavault-check_btrfs_errors` also requires such (or another) change.
